### PR TITLE
Removed Nginx

### PR DESCRIPTION
**Pull request by mlev71**

### CODE REVIEW

File: stage/services/ors/task-definitions/wsgi.json [hunk 1]
```diff
@@ -1,49 +1,28 @@
...
-   },
-   {
-      "name": "nginx",
-      "image": "jackofsum/ors-nginx",
-      "essential": true,
-      "portMappings" : [
-         {
-            "containerPort": 80,
-            "hostPort": 80
+      "entrypoint": ["uwsgi", "--ini", "http.ini"],
```
Comment (mfenner) [inline stage/services/ors/task-definitions/wsgi.json]: @mlev71 please use the conventions for logs we use elsewhere. Makes it easier to find the logs. The test infrastructure runs in eu-west-1.

File: stage/services/ors/task-definitions/wsgi.json [hunk 2]
```diff
@@ -1,49 +1,28 @@
 [
    {
-      "name": "wsgi-stage",
+      "name": "wsgi",
```
Comment (mfenner) [inline stage/services/ors/task-definitions/wsgi.json]: Please keep `wsgi-stage`, following the convention we use elsewhere.

File: stage/services/ors/task-definitions/wsgi.json (hunk 1 above)
Comment (mlev71) [reply stage/services/ors/task-definitions/wsgi.json]: Does this work now?

"logConfiguration": { 
         "logDriver": "awslogs",
         "options": { 
            "awslogs-group" : "/ecs/wsgi-stage",
            "awslogs-region": "eu-west-1",
            "awslogs-stream-prefix": "ecs"
            }
      }

**Comment by mfenner** [comment]
@mlev71 The container starts up, but then complains about a python syntax error. Is the same image running locally without problems? And can you share the source code?

**Pull request merged**